source/unified-test-format/unified-test-format.rst

```diff
@@ -639,6 +581,90 @@ The structure of this object is as follows:
+
+The structure of this object is as follows:
+
+  - ``id``: Required string. Unique name for this entity.
+
+  - ``events``: Required array of one or more strings, which denote the events
+    to be collected. Currently, only the following
+    `CMAP <../connection-monitoring-and-pooling/connection-monitoring-and-pooling.rst>`__
+    and `command monitoring <../command-monitoring/command-monitoring.rst>`__
+    events MUST be supported:
+
+      - PoolCreatedEvent
+      - PoolReadyEvent
+      - PoolClearedEvent
+      - PoolClosedEvent
+      - ConnectionCreatedEvent
+      - ConnectionReadyEvent
+      - ConnectionClosedEvent
+      - ConnectionCheckOutStartedEvent
+      - ConnectionCheckOutFailedEvent
+      - ConnectionCheckedOutEvent
+      - ConnectionCheckedInEvent
+      - CommandStartedEvent
+      - CommandSucceededEvent
+      - CommandFailedEvent
+
+For the specified entity name, the test runner MUST create the respective entity
+with a type of "event list", as described in `Supported Entity Types`_. If the
+entity already exists (such as from a previous `storeEventsAsEntity`_ object)
+the test runner MUST raise an error.
+
+The test runner MUST set up an event subscriber for each event named. The event
+subscriber MUST serialize the events it receives into a document, using the
+documented properties of the event as field names, and append the document to
+the list stored in the specified entity. Additionally, the following fields MUST
+be stored with each event document:
+
+  - ``name``: The name of the event (e.g. ``PoolCreatedEvent``). The name of the
+    event MUST be the name used in the respective specification that defines the
+    event in question.
+  - ``observedAt``: The floating-point time since the Unix epoch when the event
+    was observed by the test runner.
+
+The test runner MAY omit the ``command`` field for CommandStartedEvent and
+``reply`` field for CommandSucceededEvent.
+
+If an event field in the driver is of a type that does not directly map to a
+BSON type (e.g. ``Exception`` for the ``failure`` field of CommandFailedEvent)
+the test runner MUST convert values of that field to one of the BSON types. For
+example, a test runner MAY store the exception's error message string as the
+``failure`` field of CommandFailedEvent.
+
+If the specification defining an event permits deviation in field names, such as
+``connectionId`` field for CommandStartedEvent, the test runner SHOULD use the
+field names used in the specification when serializing events to documents even
+if the respective field name is different in the driver's event object.
+
+
+serverApi
+~~~~~~~~~
+
+Declares an API version for a `client entity <entity_client_>`_.
+
+The structure of this object is as follows:
+
+- ``version``: Required string. Test runners MUST fail if the given version
+  string is not supported by the driver.
+
+  Note: the format of this string is unrelated to `Version String`_.
```

Comment:
The Versioned API spec would already make this clear, but I figure this might avoid some confusion since we talk about version strings elsewhere in this spec.